src/main/java/com/teragrep/pth_06/scheduler/Batch.java

```diff
@@ -86,8 +88,18 @@ public Batch processRange(Offset start, Offset end) {
         LOGGER.debug("processRange");
 
         BatchSliceCollection slice = null;
```

Comment:
i would also make a new issue for this, should get rid of nulls

Reply:
Solved in commit 6eeed4b26341211fb2ef0ae491907501bcf0d929